src/common/transformations/src/transformations/init_node_info.cpp

```diff
@@ -8,9 +8,48 @@
 #include <vector>
 
 #include "itt.hpp"
+#include "openvino/core/layout.hpp"
+#include "openvino/core/preprocess/input_tensor_info.hpp"
 #include "openvino/op/util/sub_graph_base.hpp"
+#include "openvino/pass/constant_folding.hpp"
+#include "transformations/common_optimizations/remove_concat_zero_dim_input.hpp"
+#include "transformations/rt_info/decompression.hpp"
+#include "transformations/rt_info/disable_fp16_compression.hpp"
 #include "transformations/rt_info/fused_names_attribute.hpp"
+#include "transformations/rt_info/old_api_map_element_type_attribute.hpp"
+#include "transformations/rt_info/old_api_map_order_attribute.hpp"
+#include "transformations/rt_info/preprocessing_attribute.hpp"
 #include "transformations/rt_info/primitives_priority_attribute.hpp"
+#include "transformations/rt_info/transpose_sinking_attr.hpp"
+
+namespace {
+#undef TYPE_INFO
+#define TYPE_INFO(name) ov::name::get_type_info_static()
+const std::unordered_set<std::string> RtKeys = {
```

Comment:
suggest to use the following scheme to avoid global variables in library load stage:
```c++
const T& get_t() {
    static T t;
    return t;
}
```
and users should call `get_t()` to explicitly define order of global vars initialization, otherwise we can have issues with static build.

Good example is https://github.com/openvinotoolkit/openvino/pull/19976